frontend/src/styles/tokens.css

```
@@ -219,58 +187,40 @@
-     BREAKPOINT VARIABLES
-     ============================================================================ */
-
+  /* Breakpoints */
   --breakpoint-sm: 640px;
   --breakpoint-md: 768px;
   --breakpoint-lg: 1024px;
   --breakpoint-xl: 1280px;
   --breakpoint-2xl: 1536px;
-
-  /* Native control highlight */
   accent-color: var(--color-primary);
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)